docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -62,67 +63,41 @@ overloading.
+| `=` Assignment                                                                                                 |
+| `^` Complementing and Bitwise XOR                                                                              |
+| `&` Address-of and Bitwise AND                                                                                 |
+| `                                                                                                              | ` Bitwise OR |
+| `<<` Arithmetic and Logical Left-shift                                                                         |
+| `>>` Arithmetic and Logical Right-shift                                                                        |
+| `==` Equality or equal to                                                                                      |
+| `!=` Inequality or not equal to                                                                                |
+| `>` Greater than                                                                                               |
+| `>=` Greater than or equal to                                                                                  |
+| `<` Less than                                                                                                  |
+| `<=` Less than or equal to                                                                                     |
+| `->` Return type and indirect member access                                                                    |
+| `=>` Match syntax                                                                                              |
+| `[` Subscript and deduced parameter lists                                                                      |
+| `]` Subscript and deduced parameter lists                                                                      |
+| `(` Function call, function declaration and tuple literals                                                     |
+| `)` Function call, function declaration and tuple literals                                                     |
+| `{` Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `}` Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `,` Separate tuple and struct elements                                                                         |
+| `.` Member access                                                                                              |
+| `:` Name bindings                                                                                              |
+| `;` Statement separator                                                                                        |
+| `:!` Type-checking                                                                                             |
```

Comment:
I think "Generic binding" would be more accurate. This should probably go next to `:`, since the two are very closely related.